include/net/fota_download.h

```diff
@@ -146,6 +146,28 @@ int fota_download(const char *host, const char *file, const int *sec_tag_list,
 		  uint8_t sec_tag_count, uint8_t pdn_id, size_t fragment_size,
 		  const enum dfu_target_image_type expected_type);
 
+/**@brief Download the given file with the specified image type from the given host.
+ *
+ * Validate that the file type matches the expected type before proceeding with the download.
+ * When the download is complete, the secondary slot of MCUboot is tagged as having
+ * valid firmware inside it. The completion is reported through an event.
+ *
+ * URI parameters (host and file) are not copied, so pointers must stay valid
+ * until download is finished.
+ *
+ * @param host Name of host to start downloading from. Can include scheme
+ *             and port number, for example https://google.com:443
+ * @param file Path to the file you wish to download. See fota_download_any()
+ *             for details on expected format.
```

Comment:
```suggestion
 * @param host URI of the host to connect to.
 *                          Can include scheme, port number and full file path, defaults to
 *                          HTTP or HTTPS if no scheme is provided.
 * @param file   File to download or NULL if path is already provided in host parameter.
```

Follow the same documentation as `download_client_get()`. So full URI is OK to be fed in the `host` parameter.

Reply:
Notably existing fota_download() does not allow `file==NULL`